src/Microsoft.Security.Utilities.Core/UrlCredentials.cs

```diff
@@ -12,7 +12,7 @@ public UrlCredentials()
     {
         Id = "SEC101/127";
         Name = nameof(UrlCredentials);
-        Label = "a URL with login credentials";
+        Label = Resources.Label_UrlCredentials;
```

Comment:
Looks like you got tripped up with an existing inconsitency in the code base. Can you rename this file to SEC101_127_UrlCredentials.cs and then rename the label to Label_SEC101_127_UrlCredentials?